app/src/lib/menu-update.ts

```diff
@@ -162,14 +168,18 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     menuStateBuilder.disable('view-repository-on-github')
     menuStateBuilder.disable('push')
     menuStateBuilder.disable('pull')
+
+    menuStateBuilder.disable('new-repository')
+    menuStateBuilder.disable('add-local-repository')
+    menuStateBuilder.disable('clone-repository')
```

Comment:
So I don't think this is quite right either (if I'm reading this correctly) since we're still disabling these items when there's no repository selected and we shouldn't.